### PR TITLE
Design and handlebars

### PR DESCRIPTION
**Pull request by joeyswafford**

### CODE REVIEW

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-  <div class="flex-column justify-space-around">
-
-    <header class="display-flex justify-space-between align-center p-2">
-
-      <h1>
-        <a href="/">One Man's Trash</a>
-      </h1>
+  {{!-- <div class="flex-column justify-space-around"> --}}
+    <div>
+    {{!-- <header class="display-flex justify-space-between align-center p-2">
```
Comment (joeyswafford) [inline views/layouts/main.handlebars:19]: todo: will remove comments.

File: views/partials/header.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<div class="columns is-mobile">
+    <div class="column is-3">
+      <div class="navbar h-170 rounded ml-5 mt-5 is-dark">
+        <div class="container has-text-weight-medium is-size-4">
+          One Man's Trash
+        </div>
+      </div>
+    </div>
```
Comment (joeyswafford) [inline views/partials/header.html:8]: todo: will delete unnecessary html file.

File: views/partials/specifiedCategory.handlebars [hunk 3]
```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
```
Comment (joeyswafford) [inline views/partials/specifiedCategory.handlebars:1]: todo: will put this file that is unrelated to this pr in a different branch.

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -6,41 +6,18 @@
...
-
-      <nav>
-        <!-- Conditionally render login or logout links -->
-        {{#if logged_in}}
-        <a href="/profile">profile</a> |
-        <button class="no-button" id="logout">logout</button>
-        {{else}}
-        <a href="/login">login</a>
-        {{/if}}
-      </nav>
```
Comment (joeyswafford) [inline views/layouts/main.handlebars:31]: Removed this in order to just see the design without anything potentially messing it up. Will put back in but comment it out so code is there but not affecting the view.

File: views/partials/header.handlebars [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+<div class="columns">
+  <div class="column is-3">
+    <div class="navbar h-170 rounded ml-5 mt-5 is-dark" style="background-color: #4c4c47">
+      <div class="column has-text-centered has-text-weight-medium is-size-3">
+        One Man's Trash
+      </div>
+    </div>
+  </div>
+  <div class="column">
+    <nav class="navbar h-90 rounded is-dark mr-5 mt-5 ml-5" style="background-color: #4c4c47" role="navigation"
```
Comment (joeyswafford) [inline views/partials/header.handlebars:11]: @joeyswafford remove inline style to css file.

File: views/partials/shelf.handlebars [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<div class="columns b-bot mb-105 mx-5 relative">
+    
+    <p class="shelf-header mb-4 ml-2r is-size-4">{{title}}</p>
+    {{#each products}}
+    {{> product}}
+    {{/each}}
+    
+    {{!-- {{> product description='something' price='free'}}
+    {{> product description='everything' price='free'}}
+    {{> product description='idk' price='free'}} --}}
```
Comment (joeyswafford) [inline views/partials/shelf.handlebars:10]: @joeyswafford

File: views/partials/sideNav.handlebars [hunk 7]
```diff
@@ -0,0 +1,17 @@
+<!-- SIDENAV -->
+<div class="">
+  <input class="is-dark ml-5 pl-5 rounded border w-95 h-35" type="text" placeholder="Search..." />
+  <i class="fas fa-search" aria-hidden="true" style="position: absolute; left: 42px; top: 21px"></i>
```
Comment (joeyswafford) [inline views/partials/sideNav.handlebars:4]: @joeyswafford remove inline styling and add css class.

File: views/partials/sideNav.handlebars [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+<div class="filters ml-5 mt-5">
+  <h2 class="filterText b-bot">FILTERS</h2>
+  <h2 class="location">CARY, NC 27519</h2>
+</div>
+
+<div class="has-text-centered">
+  <div class="category ml-5 mt-5">
+    <h2 class="whichCategory is-size-4 b-bot">SHOP BY CATEGORY</h2>
+  </div>
+  <div class="navbar rounded ml-5 mt-5 is-dark h-1185" style="background-color: #4c4c47"></div>
```
Comment (joeyswafford) [inline views/partials/sideNav.handlebars:16]: @joeyswafford same ^

**Review by WillBerner — APPROVED** [review]
LGTM

File: server.js [hunk 9]
```diff
@@ -47,6 +47,8 @@ app.set('view engine', 'handlebars');
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, 'views')));
```
Comment (WillBerner) [inline server.js:50]: What effect does this have? I commented it out after pulling down the changes and can't notice a difference.
